src/main/java/de/tum/cit/aet/artemis/core/FilePathType.java

```diff
@@ -0,0 +1,7 @@
+package de.tum.cit.aet.artemis.core;
+
+public enum FilePathType {
```

Comment:
Can you add docs about naming conventions, ordering, etc.
A blob like this is scary to touch in future iterations.